### PR TITLE
feat: enabled cors

### PR DESCRIPTION
**Pull request by waynewee**
attempt #1 to enable

### CODE REVIEW

**Comment by Nebulis** [comment]
Hum why?

**Comment by Nebulis** [comment]
What do we need cors for

**Comment by waynewee** [comment]
> What do we need cors for

we want to enable cors on the dns sandbox to create the temporary dns on the trade trust demo flow. Currently gives a CORS error when attempting to get the executionId. Its under this story --> https://www.pivotaltracker.com/n/projects/2424361/stories/178479425

**Comment by Nebulis** [comment]
Ok then limit it to tradetrust domain

File: getExecutionDetails.ts [hunk 1]
```diff
@@ -11,6 +11,12 @@ interface GetNameEvent {
   pathParameters: { executionId: string };
 }
 
+const headers = {
+  "Content-Type": "application/json",
+  "Access-Control-Allow-Origin": "https://dev.tradetrust.io",
```
Comment (Nebulis) [inline getExecutionDetails.ts:16]: don't need for tradetrust.io ?

Comment (waynewee) [reply getExecutionDetails.ts:16]: only dev.tradetrust.io bc the demo flow only works for ropsten

**Review by Nebulis — APPROVED** [review]

**Pull request merged**